addons/policies/bootstrap_entity_policies.json

```diff
@@ -3298,6 +3298,115 @@
                     "entity-delete"
                 ]
             }
+        },
+        {
+            "typeName": "AuthPolicy",
+            "attributes":
+            {
+                "name": "AI_CLASSIFICATION",
+                "qualifiedName": "AI_CLASSIFICATION",
+                "description": "Allows tagging AI assets",
+                "policyCategory": "bootstrap",
+                "policySubCategory": "default",
+                "policyServiceName": "atlas",
+                "policyType": "allow",
+                "policyPriority": 0,
+                "policyUsers":
+                [],
+                "policyGroups":
+                [],
+                "policyRoles":
+                [
+                    "$admin",
+                    "$api-token-default-access"
+                ],
+                "policyResourceCategory": "ENTITY",
+                "policyResources":
+                [
+                    "entity-type:AIApplication",
+                    "entity-type:AIModel",
+                    "entity-classification:*",
+                    "entity:*",
+                    "classification:*"
+                ],
+                "policyActions":
+                [
+                    "entity-add-classification",
+                    "entity-update-classification",
+                    "entity-remove-classification"
+                ]
+            }
+        },
+        {
+            "typeName": "AuthPolicy",
+            "attributes":
+            {
+                "name": "READ_AI_APPLICATION_ENTITIES",
+                "qualifiedName": "READ_AI_APPLICATION_ENTITIES",
+                "description": "Allows user to perform read operation on AI application assets",
+                "policyCategory": "bootstrap",
+                "policySubCategory": "default",
+                "policyServiceName": "atlas",
+                "policyType": "allow",
+                "policyPriority": 0,
+                "policyUsers":
+                [],
+                "policyGroups":
+                [],
+                "policyRoles":
+                [
+                    "$member",
+                    "$guest"
```

Comment:
Check if you also need `"$admin", "$api-token-default-access"` here. Refer this READ_GLOSSARY policy for example -- https://github.com/atlanhq/atlas-metastore/blob/777aa9a76596d927946c898c10082756fb9cbfaf/addons/policies/bootstrap_entity_policies.json#L319

Not sure why `READ_DATA_MESH_ENTITIES` doesn't have these roles.

Reply:
Probably because domain have another policy providing read access to these roles -- https://github.com/atlanhq/atlas-metastore/blob/777aa9a76596d927946c898c10082756fb9cbfaf/addons/policies/bootstrap_entity_policies.json#L2875